commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy

```diff
@@ -17,28 +17,28 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
 
-import org.exoplatform.webui.core.model.SelectItemCategory ;
-import org.exoplatform.webui.core.model.SelectItemOption ;
-  
-  List templates = new ArrayList() ;
-  
-  SelectItemCategory row = new SelectItemCategory("row") ; 
-    row.addSelectItemOption(new SelectItemOption("oneRow",
-        "<container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\"></container>",
-        "OneRowContainerLayout"));
-    
-     row.addSelectItemOption(new SelectItemOption("twoRows",
-        "<container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\">" +
-        "  <container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\"></container>" +
-        "  <container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\"></container>" +
-        "</container>",
-        "TwoRowContainerLayout")) ;
-     row.addSelectItemOption(new SelectItemOption("threeRows",
-        "<container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\">" +
-        "  <container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\"></container>" +
-        "  <container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\"></container>" +
-        "  <container template=\"system:/groovy/portal/webui/container/UIContainer.gtmpl\"></container>" +          
-        "</container>",
+import org.exoplatform.webui.core.model.SelectItemCategory ;
```

Comment:
keep the same indentation to see the diff

Reply:
done !